samples/SampleHostPage.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+
+    <!-- Enable IE Standards mode -->
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <style type="text/css">
+        #office_form {
+            width: 100%;
+            height: 100%;
+            position: absolute;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            margin: 0;
+            border: none;
+            display: block;
+        }
+    </style>
+</head>
+<body style="margin: 0; padding: 0; overflow:hidden;">
```

Comment:
I was trying to keep the sample to a single file. You think a comment in the HTML pointing out that in a real page the styles should be separate is enough, or should I just create a CSS file and move the styles there?